BeatSaberHTTPStatus/Plugin.cs

```diff
@@ -547,13 +561,24 @@ internal class Plugin : ICutScoreBufferDidFinishEvent {
 			statusManager.EmitStatusUpdate(ChangedProperties.Beatmap, "resume");
 		}
 
+		public void OnNoteWasSpawned(NoteController noteController) {
+			NoteData noteData = noteController.noteData;
+
+			noteMapping.Add(noteData, noteController);
+
+			SetNoteDataStatus(noteData);
+
+			statusManager.EmitStatusUpdate(ChangedProperties.PerformanceAndNoteCut, "noteSpawn");
```

Comment:
It makes very little sense to be sending performance data here. Most of the note cut related properties will also have nonsensical values, as no cut actually happened. Ideally we wouldn't send any of the properties which would make no sense, such as `saberDir` or `swingRating`, and include the appropriate information in `protocol.md`.

I'm aware that this will make little sense as all of it will still exist in an object property called `noteCut`, but that's just one of those use cases I did not anticipate when initially writing this mod. If you have better ideas that don't break backwards compatibility, feel free to suggest them.

Reply:
Added ChangedProperties.NoteCut, assuming that's what you're asking.